py/selenium/webdriver/support/wait.py

```diff
@@ -16,15 +16,18 @@
 # under the License.
 
 import time
+import typing
+
+from selenium.types import WaitExcTypes
 from selenium.common.exceptions import NoSuchElementException, InvalidSelectorException
 from selenium.common.exceptions import TimeoutException
 
-POLL_FREQUENCY = 0.5  # How long to sleep in between calls to the method
-IGNORED_EXCEPTIONS = (NoSuchElementException,)  # exceptions ignored during calls to the method
+POLL_FREQUENCY: float = 0.5  # How long to sleep in between calls to the method
+IGNORED_EXCEPTIONS: typing.Tuple[typing.Type[Exception]] = (NoSuchElementException,)  # default to be ignored.
 
 
 class WebDriverWait:
-    def __init__(self, driver, timeout, poll_frequency=POLL_FREQUENCY, ignored_exceptions=None):
+    def __init__(self, driver, timeout: float, poll_frequency: float = POLL_FREQUENCY, ignored_exceptions: typing.Optional[WaitExcTypes] = None):
```

Comment:
on my prios to get black into `tox` to have the line length problems automatically handled etc